cabal.project

```diff
@@ -1,3 +1,9 @@
+packages:
+         .
```

Comment:
Does the `.` have to be on the following line to enable parsing this by `flake.nix`?

(Anyway, `packages: .` is the default so I don't know why it is here in the first place.  Could maybe be deleted, but then posterity could add it again in a way that `flake.nix` does not understand...)

Reply:
Yes, I believe it is a bug https://github.com/srid/haskell-flake/issues/193

Reply:
Seems like the parser there wants this layout:
```
packages:
  foo
  bar
```

Reply:
Interestingly I think the error still occurs with only two spaces:
```
packages:
  .
```
I checked yesterday.

Reply:
improved, it works with two spaces, thanks